src/components/BookLists.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { delBooks } from '../redux/books/booksSlice';
+
+const BooksList = ({ bookItem }) => {
+  const dispatch = useDispatch();
+  const removeBook = (id) => {
+    dispatch(delBooks(id));
+  };
+  return (
+    <li>
+      <h1>{bookItem.title}</h1>
+      <h3>{bookItem.author}</h3>
+      <button type="button" onClick={() => removeBook(bookItem.item_id)}>Delete</button>
```

Comment:
- [ Required ] I think it would be better if you fix the error that occurs when deleting the books a user has added to the book list. I've noticed a user can add as many books as possible but when I try to delete a single book that i've added, all the books I added to the list are deleted. I think this is because these new books share the same `item_id`, please see the video below and rectify this.

## Video
https://github.com/ctokoli/bookstore-with-react/assets/16392046/8b7f867c-c37a-4338-97a4-58b005926146

Reply:
Thanks @Felix45 for the review and recommended solution.